Editor/Scripts/interactivetutorials_dialog.py

```diff
@@ -147,17 +168,27 @@ def load_tutorial(self, index):
 
         tutorial_factory = self.tutorials[index]["tutorial"]
         self.current_tutorial = tutorial_factory()
+        #tutorial factory creates a new instance of the tutorial
+        #self.current_tutorial : rigid body / wind forces
+        #num_steps = len(self.currenttutorial.get_steps)
+        #^ store in self.current_turoail num steps
+        #whenever updatestepview gets called use  index and string to set the text on new label
+        self.current_tutorial_num_steps = len(self.current_tutorial.get_steps())
 
         # Invoke the tutorial start method
         self.current_tutorial.on_tutorial_start()
 
         # Update the title based on the loaded tutorial
         self.setWindowTitle("InteractiveTutorials - " + self.current_tutorial.get_title())
 
+        #set counter
+        self.current_step_index = 0 
+        #wherever load next step and load previous step + 11 -1
```

Comment:
Can remove these temp comments and move the code below the `# Reset initial state...` comment below